bech32-th/ChangeLog.md

```diff
@@ -0,0 +1,5 @@
+# ChangeLog for `bech32-th`
+
+## 1.0.2 -- 2020-02-18
```

Comment:
We start immediately with `1.0.2` so that `bech32` and `bech32-th` can be kept in lockstep with one another.